### PR TITLE
Updated prereqs and removed preview info for ade-vmss-templates

### PR DESCRIPTION
**Pull request by Jyotsna-Anand**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* Added prereq folder with mandatory parameters
* Modified username to confirm with username requirements for the windows vmss template
*

### CODE REVIEW

File: 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        "subnetName": "[concat(variables('namingInfix'), 'subnet')]",
+        "loadBalancerName": "[concat(variables('namingInfix'), 'lb')]",
+        "natPoolName": "[concat(variables('namingInfix'), 'natpool')]",
+        "bePoolName": "[concat(variables('namingInfix'), 'bepool')]",
+        "keyVaultResourceID": "[resourceId(parameters('keyVaultResourceGroup'), 'Microsoft.KeyVault/vaults/', parameters('keyVaultName'))]"
+    },
+    "resources": [{
+            "type": "Microsoft.Network/virtualNetworks",
+            "name": "[variables('virtualNetworkName')]",
+            "location": "[resourceGroup().location]",
```
Comment (bmoore-msft) [inline 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json]: Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

File: 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                    }
+                }],
+                "backendAddressPools": [{
+                    "name": "[variables('bePoolName')]"
+                }],
+                "inboundNatPools": [{
+                    "name": "[variables('natPoolName')]",
+                    "properties": {
+                        "frontendIPConfiguration": {
+                            "id": "[concat(resourceId('Microsoft.Network/loadBalancers',variables('loadBalancerName')),'/frontendIPConfigurations/loadBalancerFrontEnd')]"
```
Comment (bmoore-msft) [inline 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json]: remove concat, just use resourceId()

resourceId('Microsoft.Network/loadBalancers/frontendIPConfigurations', variables('loadBalancerName'), 'loadBalancerFrontEnd')

File: 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        "publicIPAddressName": "[concat(variables('namingInfix'), 'pip')]",
+        "subnetName": "[concat(variables('namingInfix'), 'subnet')]",
+        "loadBalancerName": "[concat(variables('namingInfix'), 'lb')]",
+        "natPoolName": "[concat(variables('namingInfix'), 'natpool')]",
+        "bePoolName": "[concat(variables('namingInfix'), 'bepool')]"
+    },
+    "resources": [{
+            "type": "Microsoft.Network/virtualNetworks",
+            "name": "[variables('virtualNetworkName')]",
+            "location": "[resourceGroup().location]",
```
Comment (bmoore-msft) [inline 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json]: Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

File: 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+                    }
+                }],
+                "backendAddressPools": [{
+                    "name": "[variables('bePoolName')]"
+                }],
+                "inboundNatPools": [{
+                    "name": "[variables('natPoolName')]",
+                    "properties": {
+                        "frontendIPConfiguration": {
+                            "id": "[concat(resourceId('Microsoft.Network/loadBalancers',variables('loadBalancerName')),'/frontendIPConfigurations/loadBalancerFrontEnd')]"
```
Comment (bmoore-msft) [inline 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json]: remove concat and use resourceID

Comment (Jyotsna-Anand) [reply 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json]: Addressed these comments in commit# 9f41323.

File: 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json (hunk 1 above)
Comment (Jyotsna-Anand) [reply 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json]: Addressed these comments in commit# 9f41323.

File: 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json [hunk 5]
```diff
@@ -13,6 +13,13 @@
             "metadata": {
                 "description": "Number of desired VM instances"
             }
+        },
```
Comment (bmoore-msft) [inline 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json]: format the JSON (e.g. SHIFT+ALT+F in VS Code)

File: 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json [hunk 6]
```diff
@@ -13,6 +13,13 @@
             "metadata": {
                 "description": "Number of desired VM instances"
             }
+        },
```
Comment (bmoore-msft) [inline 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json]: format the JSON (e.g. SHIFT+ALT+F in VS Code)

Comment (Jyotsna-Anand) [reply 201-decrypt-vmss-linux/prereqs/prereq.azuredeploy.json]: Fixed in 2e2c891

File: 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json (hunk 5 above)
Comment (Jyotsna-Anand) [reply 201-encrypt-running-vmss-linux/prereqs/prereq.azuredeploy.json]: Fixed in 2e2c891

**Pull request merged**